metadetect/lsst/util.py

```diff
@@ -825,6 +825,35 @@ def get_stats_mask(exp):
     return stats_mask
 
 
+def get_detection_mask(exp):
+    """
+    Get a mask for detection, regions with these
+    flags will not be searched for objects. If BRIGHT
+    is found in the mask plane it is added to the usual
+
+    ['EDGE', 'NO_DATA']
+
+    Parameters
+    ----------
+    exp: lsst.afw.image.ExposureF
+        The exposure
+
+    Returns
+    -------
+    A list of mask planes
+    """
+
+    # these will be ignored when finding the image standard deviation
+    # stats_mask = ['BAD', 'SAT', 'EDGE', 'NO_DATA']
```

Comment:
What happened to BAD or SAT here? Leaving this comment in the code without an explanation as to why we don't use BAD or SAT makes it seem like we left them out by accident.

Reply:
that's crud, will remove